release_dashboard/test/fakes/fake_start_context.dart

```diff
@@ -0,0 +1,150 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_core/conductor_core.dart';
+
+import 'package:file/file.dart';
+import 'package:file/memory.dart';
+import 'package:platform/platform.dart';
+
+import 'fake_process_manager.dart';
+
+const String kFlutterRoot = '/flutter';
+const String kCheckoutsParentDirectory = '$kFlutterRoot/dev/tools/';
+
+class TestStdio extends Stdio {
```

Comment:
This probably shouldn't be in a file named `fake_start_context.dart`. same for the constants above.

Reply:
Moved `TestStdio`, is it ok to keep `const String kFlutterRoot = '/flutter';
const String kCheckoutsParentDirectory = '$kFlutterRoot/dev/tools/';` here though? since they are being referenced directly in this file.

Reply:
hmm, I guess this was test-related globals that are unused in this test suite. i guess it's fine to leave it here.